backend/src/graphql/resolver/util/creations.test.ts

```diff
@@ -225,8 +228,10 @@ describe('util/creation', () => {
           })
 
           it('has the clock set correctly', () => {
+            const targetMonth = nextMonthTargetDate.getMonth() + 1
+            const targetMonthString = (targetMonth < 10 ? '0' : '') + String(targetMonth)
```

Comment:
check if this is a relevant thing to avoid this ternary https://stackoverflow.com/questions/2998784/how-to-output-numbers-with-leading-zeros-in-javascript